src/ResourceManagement/ContainerRegistry/Microsoft.Azure.Management.ContainerRegistry/project.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "version": "1.1.0-preview",
+  "version": "1.2.0",
```

Comment:
Yes, we have a concept of *preview* vs *stable* SDKs: when a breaking change is introduced (or a large feature is added) to the SDK, the package must go into (or stay) *preview* for X amount of time, after which the package can be considered *stable* and the `-preview` suffix can be removed from the version. We are trying to give customers an idea of the stability of the packages they are using from this repository.